README.md

```diff
@@ -58,8 +58,33 @@ The CSI driver is configured via environmental variables, any value that doesn't
 | `X_CSI_SYSTEM_FILES`       | all        | All required storage driver-specific files archived in tar, tar.gz or tar.bz2 format|                                                                                        | /path/to/etc-ceph.tar.gz                                                                                                                                                                                                                |
 | `X_CSI_DEBUG_MODE`         | all        | Debug mode (rpdb, pdb) to use. Disabled by default.           |                                                                                                              | rpdb                                                                                                                                                                                                                                    |
 | `X_CSI_ABORT_DUPLICATES`   | all        | If we want to abort or queue (default) duplicated requests.   | false                                                                                                        | true                                                                                                                                                                                                                                    |
+
 ### Create required secrets which the EmberCSI resource will use
+
+#### Create the tar file
+
+##### Example for ceph backend
+
+- Prepare the following tree structure:
+    - etc/ceph/ceph.conf
+    - etc/ceph/keyring
+
+- Create the archive:
+    ```
+    $ tar -cvf system-files.tar etc
+    ```
+    
+Notes:
+
+- The keyring file depends on the specified client user. E.g. ceph.client.cinder.keyring
```

Comment:
I would suggest mentioning that in the above example, we are renaming the 'ceph.client.cinder.keyring' file to just 'keyring' for brevity.

Reply:
Sure. Done.